german/GUI_Automation/src/test/java/at05ui/runner/RunCukesTest.java

```diff
@@ -0,0 +1,42 @@
+package at05ui.runner;
+
+
+import at05ui.sampleapp.SampleAppAutomation;
+import cucumber.api.java.Before;
+import cucumber.api.testng.AbstractTestNGCucumberTests;
+import org.apache.log4j.Logger;
+import org.testng.annotations.AfterTest;
+import org.testng.annotations.BeforeTest;
+
+
+public class RunCukesTest extends AbstractTestNGCucumberTests {
+    private static Logger log = Logger.getLogger("RunCukesTest");
+
+    /**
+     * Starts the execution
+     */
+    @BeforeTest
+    public void beforeExecution() {
+        try {
+            //ToDo Review if this is executed only once
+            SampleAppAutomation.getInstance();//.startUp();
```

Comment:
Where do you call to startup() method?

Reply:
in the constructor of the necessary classes